src/pages/billing/List.js

```diff
@@ -427,6 +421,19 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
               p(['It may not exist, or you may not have access to it.'])
             ])
           ])],
+        [azureUserWithNoBillingProjects || creatingAzureBillingProject, () => h(AzureBillingProjectWizard, {
+          onSuccess: billingProjectName => {
+            Ajax().Metrics.captureEvent(Events.billingCreationBillingProjectCreated, {
+              billingProjectName, cloudPlatform: cloudProviders.azure.label
+            })
+            setCreatingBillingProject(null)
+            loadProjects()
+            Nav.history.push({
+              pathname: Nav.getPath('billing'),
+              search: qs.stringify({ selectedName: billingProjectName, type: 'project' })
+            })
+          }
+        })],
```

Comment:
Note that I haven't done much testing of _actually_ creating a billing project because of the overhead (and the fact that the billing project cannot be deleted now due to dangling references).

But please someone **do** test create a billing project, ideally with the developer console open so that you can look for any possible errors (I saw a React warning once about state changes after an unmounted component, but then it did not reproduce).

Reply:
Testing this out and dropping my notes here:
- email validation isn't quite right/isn't very strict. `marc.test` counts as a valid email. When I attempted to create a billing project while adding that user, Rawls returns a 409 saying `Users marc.test have not signed up for Terra` which then manifests in the wizard as an error under the billing project name that says a project by that name already exists. Not sure if the status code should be something else (maybe a 400?) or if we need to handle the error better in the UI.
-  after creating a billing project, it selects the billing project right away and allows you to add members to the billing project even though the project is still creating. I tested it out and it seems to have worked, so maybe it's fine, but it is different from Google where we don't allow you to select a creating billing project. Selecting another project leaves the creating project un-selectable again.

Everything else looked good!

Reply:
I think it's kind of confusing how it's selected initially, but then if you click away, you can't select it again. 

It might be nice to have it selectable, and then show some indication that it isn't ready yet (greyed-out tabs, a message, etc). But in the scope here, with the current page behavior, it might be better to just return to the main page with no project selected. Although I suppose maybe if you had dozens of these, that could be obnoxious in it's own right, since you would have to go hunting for it.

Reply:
I agree that we should just go back to the billing page itself with no selected project.